Library/Homebrew/utils.rb

```diff
@@ -431,6 +431,7 @@ def which_editor
 end
 
 def exec_editor(*args)
+  puts "Editing #{args.join "\n"}"
```

Comment:
This doesn't render very nicely if multiple arguments are supplied. May I suggest the following as a replacement?

```ruby
ohai "Editing:", args
```